### PR TITLE
refactor(console): update onboarding phrases

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

update onboarding phrases

## Testing

image

image

image

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts [hunk 1]
```diff
@@ -39,18 +39,18 @@ const cloud = {
     },
     reason_field: 'I’m signing up because',
     reason_options: {
-      adoption: 'Implementing new tech infra for early projects',
-      replacement: 'Seeking a replacement for the current authentication system',
-      evaluation: 'Investigating open-source or budget-friendly SaaS options',
-      experimentation: 'Exploring and experimenting with OIDC and products',
-      aesthetics: 'Finding an auth solution with beautiful UI and components',
+      passwordless: 'Finding password-less authentication and UI kit',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en/translation/admin-console/cloud.ts:42]:
```suggestion
      passwordless: 'Finding passwordless authentication and UI kit',
```

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts [hunk 2]
```diff
@@ -60,11 +60,10 @@ const cloud = {
     enter_admin_console: 'Enter Logto Cloud Preview',
   },
   gift: {
-    title: 'Secure early credit for Logto SaaS join the front-runners now !',
-    description: 'Book a one-on-one session with our team for early credit and use case sharing.',
+    title: 'Use Logto Cloud free for 3 months. Join the front-runners now !',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en/translation/admin-console/cloud.ts:63]:
```suggestion
    title: 'Use Logto Cloud free for 60 days. Join the front-runners now !',
```

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts [hunk 3]
```diff
@@ -60,11 +60,10 @@ const cloud = {
...
   },
   gift: {
-    title: 'Secure early credit for Logto SaaS join the front-runners now !',
-    description: 'Book a one-on-one session with our team for early credit and use case sharing.',
+    title: 'Use Logto Cloud free for 60 days. Join the front-runners now !',
+    description: 'Book a one-on-one session with our team for early credit.',
     reserve_title: 'Reserve your time with Logto team',
-    reserve_description:
-      'You can also save the calendar link. We’re always available to assist you.',
+    reserve_description: 'Credit is only eligible once upon evaluation. ',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en/translation/admin-console/cloud.ts:66]:
```suggestion
    reserve_description: 'Credit is only eligible once upon evaluation.',
```

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/cloud.ts]: Updated.

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts (hunk 3 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/cloud.ts]: Updated.

File: packages/phrases/src/locales/en/translation/admin-console/cloud.ts [hunk 4]
```diff
@@ -60,11 +60,10 @@ const cloud = {
     enter_admin_console: 'Enter Logto Cloud Preview',
   },
   gift: {
-    title: 'Secure early credit for Logto SaaS join the front-runners now !',
-    description: 'Book a one-on-one session with our team for early credit and use case sharing.',
+    title: 'Use Logto Cloud free for 60 days. Join the front-runners now !',
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/cloud.ts]: Updated.

**Review by gao-sun — APPROVED** [review]

**Pull request merged**